tools/admin-console/templates/healthauthority.html

```diff
@@ -49,6 +49,15 @@ <h2>Edit Health Authority '{{.ha.Issuer}}'</h2>
     </div>
   </div>
 
+  <div class="form-group row">
+    <label class="control-label col-sm-3" for="JWKS URI">JWKS URI:</label>
+    <div class="col-sm-6">
+      <input type="text" id="JwksURI" name="JwksURI" size="50" value="{{.ha.JwksURI}}">
+      <small id="NameHelpBlock" class="form-text text-muted">The JWKS URI for the heath authority.
+			  If specified, the keys will be fetched periodically.
```

Comment:
Specify how frequently?

Reply:
I didn't because it's specified in the run scripts, and therefore is overridable when deployed. I'll update the note to reflect this reality.